pkg/httpserver/httpserver_test.go

```diff
@@ -71,57 +60,89 @@ var _ = Describe("Test httpserver", func() {
 			},
 		}
 
-		httpServ := NewHTTPServer(testProbe, metricsStore, testPort, fakeDebugServer)
-
-		testServer := httptest.Server{
+		httpServ := NewHTTPServer(testProbes, nil, metricsStore, testPort, fakeDebugServer)
+		testServer = &httptest.Server{
 			Config: httpServ.server,
 		}
+		Expect(testServer).ToNot(BeNil())
```

Comment:
probably can't be nil already?
edit: i guess it can still run out of memory on the stack

Reply:
not required like you pointed out. I moved this around here to catch some test issues when the state was being shared across the tests.